google-ads-examples/src/main/java/com/google/ads/googleads/examples/errorhandling/HandlePartialFailure.java

```diff
@@ -93,8 +93,28 @@ public static void main(String[] args) throws InvalidProtocolBufferException {
   }
 
   /** Runs the example. */
-  private void runExample(GoogleAdsClient googleAdsClient, long customerId, long campaignId)
+  public void runExample(GoogleAdsClient googleAdsClient, long customerId, long campaignId)
       throws InvalidProtocolBufferException {
+    MutateAdGroupsResponse response = createAdGroups(googleAdsClient, customerId, campaignId);
+
+    // Checks for existence of any partial failures in the response.
+    if (checkIfPartialFailureErrorExists(response)) {
+      System.out.println("Partial failures occurred. Details will be shown below.");
```

Comment:
Nit: I think the introductory sentence here is useful, but "Details will be shown below" is probably not necessary.
This is because, in your printResult, you still show all the results anyway, regardless of whether the result succeeded or failed.